src/SpacesInfoCard.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  ExternalLinkIcon,
+  DocumentDuplicateIcon,
+  CodeIcon,
+  ArrowRightIcon,
+  ChevronUpIcon,
+} from "@heroicons/react/solid";
+
+const handleCopyClick = async () => {
+  const currentURL = window.location.href;
+  try {
+    await navigator.clipboard.writeText(currentURL);
+  } catch (err) {
+    console.error("Failed to copy URL: ", err);
+  }
+};
+
+const ExpandedInfoSection: FunctionComponent<{ projectInfo: ProjectInfo }> = ({
+  projectInfo,
+}) => {
+  return (
+    <div className="space-y-4">
+      <div className="text-slate-200 text-xs pb-2">
+        Open this page in multiple windows or share the URL with your team to
+        experience the demo.
+        <div>
+          <a
+            className="text-slate-200 text-xs underline"
+            href={`https://ably.com/examples/${projectInfo.topic}?utm_source=ably-labs&utm_medium=demo&utm_campaign=${projectInfo.topic}`}
+            target="_blank"
+          >
+            Learn more.
+          </a>
+        </div>
+      </div>
+      <div className="space-y-4 text-xs">
+        <a
+          className="flex justify-center bg-white rounded py-3 md:flex-grow"
+          href={window.location.href}
+          target="_blank"
+        >
+          Open in new window
+          <ExternalLinkIcon className="h-4 w-4 ml-2 text-orange-600" />
+        </a>
+        <div className="flex flex-row space-x-4 md:space-y-4 md:space-x-0 md:flex-col">
+          <button
+            className="flex justify-center bg-slate-700 rounded py-3 flex-grow text-white"
+            onClick={handleCopyClick}
+          >
+            <DocumentDuplicateIcon className="h-4 w-4 mx-2 text-slate-300" />
+            Copy Link
+          </button>
+          <a
+            className="flex justify-center bg-slate-700 rounded py-3 flex-grow text-white"
+            href={`https://github.com/ably-labs/${projectInfo.repoNameAndPath}`}
+          >
+            <CodeIcon className="h-4 w-4 mx-2 text-slate-300" />
+            View docs
+            <ExternalLinkIcon className="h-4 w-4 ml-2 text-slate-300" />
+          </a>
+        </div>
+        <a
+          className="flex justify-center bg-slate-700 text-white rounded py-3 md:flex-grow"
+          href={`https://github.com/ably-labs/${projectInfo.repoNameAndPath}`}
+        >
+          <span className="h-4 w-4 ml-2">
+            <GithubLogo />
+          </span>
+          <span className="ml-2">View source on Github</span>
+          <ExternalLinkIcon className="h-4 w-4 ml-2 text-slate-300" />
+        </a>
+      </div>
+    </div>
+  );
+};
+
+const SpacesInfoCard: FunctionComponent<{ projectInfo: ProjectInfo }> = ({
+  projectInfo,
+}) => {
+  const [expanded, setExpanded] = useState(true);
+  const [isMobile, setIsMobile] = useState(window.innerWidth < 769);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth < 768);
+    };
+    window.addEventListener("resize", handleResize);
```

Comment:
We should add `lodash.debounce` to this handler so `setIsMobile` is not called until a user finishes resizing https://lodash.com/docs/4.17.15#debounce